cypress_shared/pages/apply/show.ts

```diff
@@ -0,0 +1,19 @@
+import { TemporaryAccommodationApplication as Application } from '../../../server/@types/shared'
+import paths from '../../../server/paths/apply'
+import Page from '../page'
+
+export default class ShowPage extends Page {
```

Comment:
Don't think we should need this - we have `TaskListPage` for this page, though you'll need to give it a static visit method

Reply:
will the the ``TaskListPage instead